src/idsk/components/search-component/search-component.js

```diff
@@ -0,0 +1,38 @@
+import '../../../govuk/vendor/polyfills/Function/prototype/bind'
+import '../../../govuk/vendor/polyfills/Event' // addEventListener and event.target normaliziation
+
+function SearchComponent($module) {
+    this.$module = $module
+}
+
+SearchComponent.prototype.init = function () {
+    // Check for module
+    var $module = this.$module
+    if (!$module) {
+        return
+    }
+
+    var $searchInputs = $module.querySelectorAll('.idsk-search-component__input')
+    if (!$searchInputs) {
+        return
+    } 
+
+    $searchInputs.forEach(function ($searchInput) {
+        $searchInput.addEventListener('keyup', this.handleSearchInput.bind(this))
```

Comment:
nemusi to byt na keyup, aby sa to nespustalo po kazdom stlaceni, uplne stanic aby sa to spustilo raz, npr po onchange